grr/server/grr_response_server/gui/ui/components/human_readable_size/human_readable_size.ng.html

```diff
@@ -0,0 +1 @@
+<span>{{getHumanSize()}}</span>
```

Comment:
Please indicate that the human readable size is not exact, e.g. by prefixing it with `~`. 1025 B would be displayed as `1.0 KiB` which might be unexpected.